src/uvw/util.hpp

```diff
@@ -314,7 +320,7 @@ Addr address(F &&f, const H *handle) noexcept {
 template<typename F, typename... Args>
 std::string path(F &&f, Args... args) noexcept {
     std::size_t size = DEFAULT_SIZE;
-    char buf[size];
+    char buf[DEFAULT_SIZE];
```

Comment:
Within this method, use `size` everywhere for consistency. Do not use `DEFAULT_SIZE` to declare the buffer. Thank you.